src/views/Challenge.vue

```diff
@@ -70,6 +70,7 @@ export default {
       return this.challengeSeed(this.challengeId)
     },
     sortingOptions() {
+      let proposals = this.proposals;
```

Comment:
nit: you can avoid duplicating this and use directly `this.proposals`
nit: `;` is unnecessary